### DISCUSSION TITLE
How to enable copilot on markdown files ?

### DISCUSSION THREAD

**Discussion by anshul-raman** [LazyVim/LazyVim#564] [Q&A] [answered]
I'm using the Copilot extras provided by lazyvim: 
https://www.lazyvim.org/plugins/extras/coding.copilot 

I'm not able to enable the copilot on Mardown files. I added the following snippet in my config, but it doesn't work. 

```lua
require("copilot").setup {
  filetypes = {
    markdown = true, -- overrides default
  },
}
```

**Comment by robsonpeixoto** [comment, marked as answer]
@anshul-raman read it https://www.lazyvim.org/configuration/plugins to learn how to configure a plugin.
